actions/admin_actions.jsx

```diff
@@ -162,6 +162,15 @@ export async function uploadBrandImage(brandImage, success, error) {
     }
 }
 
+export async function deleteBrandImage(success, error) {
+    const {data, error: err} = await AdminActions.deleteBrandImage()(dispatch, getState);
+    if (data && success) {
+        success(data);
+    } else if (err && error) {
+        error({id: err.server_error_id, ...err});
+    }
+}
+
```

Comment:
We're actually moving away from this pattern into a more redux-friendly format which is dispatching an action directly.  That can be done by creating ``components/admin_console/brand_image_setting/index.js`` and from there pass `uploadBrandImage` and `deleteBrandImage` as actions into ``brand_image_setting.jsx``.  That would be not necessary for this PR but good to have.
Let me know what's your plan.  That could also be done on separate PR if you're interested.